Homework7/Group.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Homework7
+{
+    class Group
+    {
+        public string GroupName { get; }
+        public List<Unit> GroupFighters { get; }
```

Comment:
It's bad practice to use List instead of using IEnumerable where possible.
Because when you will use DB it will create a lot of problems for you.
But nevermind.
Will fix together after some lessons.